staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1794,6 +1822,33 @@ type VolumeProjection struct {
 	// serviceAccountToken is information about the serviceAccountToken data to project
 	// +optional
 	ServiceAccountToken *ServiceAccountTokenProjection `json:"serviceAccountToken,omitempty" protobuf:"bytes,4,opt,name=serviceAccountToken"`
+
+	// ClusterTrustBundle allows a pod to access the `.spec.trustBundle` field
+	// of a ClusterTrustBundle object in an auto-updating file.
+	//
+	// Alpha, gated by the ClusterTrustBundleProjection feature gate.
+	//
+	// ClusterTrustBundle objects can either be selected by name, or by the
+	// combination of signer name and a label selector.
+	//
+	// When selecting by name, the referenced ClusterTrustBundle object must
+	// have an empty spec.signerName field.
+	//
+	// When selecting by signer name, the contents of all ClusterTrustBundle
+	// objects associated with the signer and matching the label will be unified
+	// and deduplicated.  Use of a label selector is required, because signer
```

Comment:
drop the comment about the required label selector

I also think some of this doc can be dropped in favor of the field-level doc within ClusterTrustBundleProjection

Reply:
Done.